README.md

```diff
@@ -206,7 +206,8 @@ export default {
 		...
 		adapter: azure({
 			esbuildOptions: {
-				external: ['fsevents']
+				external: ['fsevents'],
+				keepNames: [true, false]
```

Comment:
Shouldn't this just be `keepNames: true` ?

Reply:
@geoffrich I don't know, honestly. My thinking is that this would give them the ability to exclusively set it to `false`, but maybe that's not applicable when by the time this flag gets to the cli (like `esbuild app.js --minify --keep-names`)

I'll concede to you; would you like me to change it?

Reply:
I think Geoff's point is just that this code block is intended to give an example, not to list all the options. As an example, `keepNames: [true, false]` is probably wrong, because the `keepNames` option only accepts a single boolean from what I understand.

Reply:
@tlaundal It is, you're right. I didn't realize Geoff's comment was on the readme.md, so I just got super confused, sorry about that

Reply:
Resolving convo since this was in [1cee492](https://github.com/geoffrich/svelte-adapter-azure-swa/pull/150/commits/1cee49287deebff8415e681255f737355cf3e6d3)